data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -1,3 +1,5 @@
+from utils.tools import parse_bcl_convert_output
```

Comment:
Need move this import after `# ---` line, please

Reply:
Fixed in 1cb94eb